pkg/workflows/wasm/host/module_test.go

```diff
@@ -22,7 +23,7 @@ type mockMessageEmitter struct {
 	labels map[string]string
 }
 
-func (m *mockMessageEmitter) Emit(msg string) error {
+func (m *mockMessageEmitter) Emit(ctx context.Context, msg string) error {
```

Comment:
change type on `e` to accept a context then in the unit tests you can make assertions on the context